rclrs/src/node/graph.rs

```diff
@@ -458,14 +455,31 @@ fn convert_names_and_types(
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::Context;
+    use crate::{Context, InitOptions};
 
     #[test]
     fn test_graph_empty() {
-        let context = Context::new([]).unwrap();
+        let domain_id: usize = std::env::var("ROS_DOMAIN_ID")
+            .ok()
+            .and_then(|value| value.parse().ok())
+            .map(|value: usize| {
+                if value == 99 {
+                    // The default domain ID for this application is 99, which
+                    // conflicts with our arbitrarily chosen default for the
+                    // empty graph test. Therefore we will set the empty graph
+                    // test domain ID to 98 instead.
```

Comment:
Hopefully this will make the situation more clear, especially for future maintainers who didn't know about this problem we were having: [21d3b35](https://github.com/ros2-rust/ros2_rust/pull/386/commits/21d3b35533ed1839a3d701074ab54672af53ca61)